04-behavr.Rmd

```diff
@@ -175,6 +175,18 @@ dt[meta = TRUE]
 
 `paste()` is a function that link strings of characters with an arbitrary separator (`"|"` here).
 
+We can also add new metavariables from a summary, in this example we compute the average time moving per fly and then we add it to the metadata.
+```{r}
+# the average time spent moving per 1000 (rounded)
+mean_mov_dt <- dt[, .(mean_moving = round(mean(moving) * 1000)), by=id]
```

Comment:
good idea to add this explanation in this section, but there is no `moving` variable in this dataset. That is why travis failed.
Could you **use mean activity**?
Also, it makes sense to add it in the end of the "Summarise data" section. this way, if follows a summary. you save lines of code.